nightguard/Utils.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Utils.swift
+//  nightguard
+//
+//  Created by Florian Preknya on 11/14/18.
+//  Copyright © 2018 private. All rights reserved.
+//
+
+import Foundation
+
+// todo: move this to another file (Tools or something similar)
+func dispatchOnMain(_ closure: @escaping () -> ()) {
+    if Thread.isMainThread {
```

Comment:
Great to change the behavior depending on the current thread. I had a lot of trouble here, before!